INSTALL.md

```diff
@@ -57,8 +57,27 @@ Shogun is part of [homebrew-science](https://github.com/Homebrew/homebrew-scienc
     $ sudo brew install shogun
 
 ###Windows <a name="windows"></a>
-Shogun natively compiles under Windows using MSVC, see the [CI build](https://ci.appveyor.com/project/vigsterkr/shogun). We currently do not support a binary
-installer. If you are interested in packaging, documenting, or contributing otherwise, please contact us.
+We currently do not support a Shogun binary on Windows platform. Shogun natively compiles under Windows using MSVC. It is recommended to use " Visual Studio 14 2015 " or " MSBuild " to build Shogun which both have good support to C++11. 
+
+Run CMD as administrator with the following commands：
+```
+git clone https://github.com/shogun-toolbox/shogun.git C:\projects\shogun 
```

Comment:
ah yeah ok. I just dont like this triple quotation marks to make markdown render it as code